Mathlib/Algebra/Algebra/Subalgebra/Centralizer.lean

```diff
@@ -59,13 +59,13 @@ open Finsupp TensorProduct
 
 /--
 Let `R` be a commutative ring and `A, B` be `R`-algebras where `B` is free as `R`-module.
-For any subalgebra `S` of `A`, the centralizer of `S ⊆ A ⊗ B` is `C_S(A) ⊗ B` where `C_S(A)` is the
+For any subset `S ⊆ A`, the centralizer of `S ⊆ A ⊗ B` is `C_S(A) ⊗ B` where `C_S(A)` is the
```

Comment:
```suggestion
For any subset `S ⊆ A`, the centralizer of `S ⊆ A ⊗ B` is `C_A(S) ⊗ B` where `C_A(S)` is the
```
Also, `S ⊗ 1 ⊆ A ⊗ B` would be clearer IMO. If you change it please also change the other occurrences.